src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -553,7 +549,10 @@
         <Exclude Condition="'%(FilePackageDependency.IsReferenceAsset)' != 'true'">Compile</Exclude>
       </FilePackageDependency>
     </ItemGroup>
-      
+    
+    <Error Condition="'@(BaseLinePackage)' == ''" Text="The BaseLinePackage item is not defined: ensure you have imported Microsoft.Private.PackageBaseLine.props from the Microsoft.Private.PackageBaseLine package" />
```

Comment:
I didn't want folks to miss this.  I suppose I could condition them better.

Can you remind me why you wouldn't want the baselining to happen in CLI?  I would imagine that solves some issues for you (eg: maintaining version list).